src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc),
+    url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+/* #region search*/
+.search {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)